src/webapp/src/firebase.ts

```diff
@@ -0,0 +1,14 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+firebase.initializeApp({
```

Comment:
Ideally here we would like to pass the key as an environment variable. Can you read a little about github secrets?